WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -660,6 +725,62 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    private fun loadPostFromRemote(localPostId: Int) {
+        // We need network connection to load a remote post
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post != null) {
+            originalPostCopyForConflictUndo = post.clone()
+            dispatcher.dispatch(PostActionBuilder.newFetchPostAction(RemotePostPayload(post, site)))
+            _toastMessage.postValue(ToastMessageHolder(R.string.toast_conflict_updating_post, Duration.SHORT))
+        } else {
+            localPostIdForConflictResolutionDialog = null
+        }
+    }
+
+    private fun savePostFromLocal(localPostId: Int) {
+        // We need network connection to push local version to remote
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post == null) {
+            return
+        }
+
+        localPostIdForConflictResolutionDialog = null
```

Comment:
Similar to my comment in `loadPostFromRemote`, if we end up setting `localPostIdForConflictResolutionDialog` to `null` in `onPositiveClickedForBasicDialog`, I believe we can simply remove this one.

Reply:
👍 9a38333dcd